pom.xml

```diff
@@ -11,7 +11,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.580.1</version>
+        <version>2.60.3</version>
```

Comment:
Not even buildable. See: https://github.com/jenkinsci/plugin-pom

Reply:
done.